doc/test-suite.py

```diff
@@ -77,30 +75,39 @@
                 continue
 
             # reset the options
-            options = {**defaultOptions, **testOptions}
+            options = testOptions.copy()
 
             # filenames (input MEI/XML and output SVG)
             inputFile = os.path.join(path1, item1, item2)
-            print(inputFile)
+            options.update({"xmlIdSeed": int(hashlib.sha256(
+                inputFile.encode("utf-8")).hexdigest(), 16) % 10**9})
+            print(f'Rendering {item2}')
             name, ext = os.path.splitext(item2)
             svgFile = os.path.join(path2, item1, name + '.svg')
             pngFile = os.path.join(path2, item1, name + '.png')
+            timeMapFile = os.path.join(path2, item1, name + '.json')
```

Comment:
Not a big deal, but I find using `f` strings a bit nicer than concatenating them... 

```suggestion
            timeMapFile = os.path.join(path2, item1, f"{name}.json")
```

Reply:
Plan for the next step is to switch to pathlib and clean up the whole section. So I'll leave this for now.